.github/workflows/e2e.yaml

```diff
@@ -66,8 +66,10 @@ jobs:
       with:
         ref: ${{ inputs.checkout_ref }}
     
-    - if: ${{ !(needs.test.status == 'success')}}
-      run: exit 1 # will make status show as failure
+    - if: ${{ !(needs.test.conclusion == 'success')}}
+      run: |
+        echo "job status: ${{ needs.test.conclusion }}"
```

Comment:
there's something bugged about the existing logic but this isn't the fix. revert and we will fix in another pr.

this isn't the correct syntax for doing this. You'd use an environment variable to get the github variable passed to the bash